apps/test/util/wrappedEventListener.js

```diff
@@ -10,15 +7,22 @@ module.exports = {
    * Attach our wrapper, tracking all added listeners
    */
   attach: function () {
-    if (original) {
+    if (window.addEventListener.toString().indexOf('[native code]') === -1) {
```

Comment:
Might be worth pulling this out into an "isNativeCode" helper, just to make it clearer what's going on here.